src/app/editor/editor.component.spec.ts

```diff
@@ -16,7 +16,13 @@ describe('EditorComponent', () => {
   beforeEach(() => {
     TestBed.configureTestingModule({
       schemas: [NO_ERRORS_SCHEMA],
-      imports: [MatMenuModule, MatSnackBarModule, NgxsModule.forRoot([UiState]), RouterTestingModule.withRoutes([]),  TranslateModule.forRoot()],
+      imports: [
+        MatMenuModule,
+        MatSnackBarModule,
+        NgxsModule.forRoot([UiState]),
+        RouterTestingModule.withRoutes([]),
+        TranslateModule.forRoot()
```

Comment:
Should this be `forRoot` or should we just include the module here?

Reply:
I left this as it was before. But should not this be:
```typescript
TranslateModule.forRoot({
      loader: { provide: TranslateLoader, useClass: FakeLoader },
    })
```

Or something?

Reply:
Fixed.